src/HotChocolate/Data/src/EntityFramework/Extensions/EntityFrameworkObjectFieldDescriptorExtensions.cs

```diff
@@ -48,6 +48,14 @@ public static class EntityFrameworkObjectFieldDescriptorExtensions
                         var index = d.MiddlewareComponents.IndexOf(placeholder);
                         d.MiddlewareComponents[index] = Create(middleware);
                     }
+                    else if (d.ResultType is not null &&
```

Comment:
This is not right since the executable could also be created in one of the middleware... So maybe we always check for it?